packages/components/psammead-media-indicator/src/index.jsx

```diff
@@ -38,19 +38,40 @@ const TimeDuration = styled.time`
   margin: 0 ${GEL_SPACING_HLF};
 `;
 
-const MediaIndicator = ({ datetime, duration, type, topStory, service }) => (
-  <MediaIndicatorWrapper
-    aria-hidden="true"
-    topStory={topStory}
-    service={service}
-  >
-    <FlexWrapper>
-      {mediaIcons[type]}
-      {duration && datetime && (
-        <TimeDuration dateTime={datetime}>{duration}</TimeDuration>
-      )}
-    </FlexWrapper>
-  </MediaIndicatorWrapper>
+const IndexAlsosMediaIndicator = styled.span`
+  & > svg {
+    margin: 0;
+  }
+`;
+
+const MediaIndicator = ({
+  datetime,
+  duration,
+  type,
+  topStory,
+  service,
+  indexAlsos,
+}) => (
```

Comment:
I think you can get away without Fragments if you do something like
```suggestion
}) => indexAlsos ? (
```
(and obviously adjust the rest of the component

Reply:
Yep, for some reason it was complaining when I initially did it without Fragment. Maybe because I was wrapping it between brackets `=> { indexAlsos ? (`?

Reply:
¯\_(ツ)_/¯ looks good now